src/modules/AutoinstScripts.rb

```diff
@@ -227,7 +227,7 @@ def Export
     def Resolve_ws(script)
       script = deep_copy(script)
       if !Builtins.isempty(Ops.get_string(script, "location", ""))
-        l = CutWhitespace(Ops.get_string(script, "location", ""))
+        l = Ops.get_string(script, "location", "").strip
```

Comment:
Yes. In fact I originally rewrote the whole method, but then decided I was shaving the yak way too far and needed to focus on the original feature. :)